lib/ui/views/profile/edit_profile_view.dart

```diff
@@ -24,6 +26,19 @@ class _EditProfileViewState extends State<EditProfileView> {
   String _name, _educationalInstitute, _country;
   bool _subscribed;
 
+  final String _countriesURL = 'https://restcountries.eu/rest/v2/all';
+  final String _instituteURL = 'http://universities.hipolabs.com/search?';
```

Comment:
Would be better if we keep the API endpoints in its API class.

Reply:
I have used same API class for both `countries` and `institutes`. To keep the endpoints in the API class I will have to split them into two separate classes . So do you want me to separate them?

Reply:
No what you can do is define constants which will used to switch between countries and institutes